build.gradle

```diff
@@ -2,6 +2,8 @@
 apply from: 'dependencies.gradle'
 
 buildscript {
+    apply from: 'dependencies.gradle'
```

Comment:
Fun fact: gradle will not find vars defined in an external file when referring to them in the buildscript block, unless you link it from the buildscript block, too. I'm not sure about the "fun" part though.